core/signeddata.go

```diff
@@ -691,6 +692,56 @@ func (s SignedRandao) clone() (SignedRandao, error) {
 	return resp, nil
 }
 
+// NewPartialSignedBeaconCommitteeSubscription is a convenience function which returns new partially signed BeaconCommitteeSubscription.
+func NewPartialSignedBeaconCommitteeSubscription(sub eth2exp.BeaconCommitteeSubscription, shareIdx int) ParSignedData {
```

Comment:
wondering if we can drop the `Beacon` part without loss of context?

Reply:
i suggest we should keep this to align with underlying type